lib/liveValidation/liveValidator.ts

```diff
@@ -844,7 +844,7 @@ export const parseValidationRequest = (
     // Lower all the keys and values of query parameters before searching for `api-version`
     const queryObject: ParsedUrlQuery = {};
     parsedUrl.searchParams.forEach((value, key) => {
-      queryObject[key.toLowerCase()] = value;
+      queryObject[key.toLowerCase()] = value.toLowerCase();
```

Comment:
Can you check type of value before calling `tolowercase`?  
```
_.isString(value) ? value.toLowerCase() : value
```

Reply:
Search Parameter value must be string. We do not need additional check.

Reply:
```    forEach(callback: (value: string, name: string, searchParams: this) => void): void;``` method signiture.